backend/src/main/java/ai/verta/modeldb/comment/CommentServiceImpl.java

```diff
@@ -168,6 +180,10 @@ public void getExperimentRunComments(
                 .build();
         throw StatusProto.toStatusRuntimeException(status);
       }
+      String projectId = experimentRunDAO.getProjectIdByExperimentRunId(request.getEntityId());
```

Comment:
with shadow delete direct access to comments gave access to comments, to put the call behind auth needed this change